### PR TITLE
server/gnocco: make revive happy

### LINKED ISSUE #7: Make revive happy
At the moment revive is not that happy about Gnocco:

```sh
⚠  https://revive.run/r#package-comments  should have a package comment
server/gnocco/cache.go:1:1
⚠  https://revive.run/r#exported  exported function NewFromFilename should have comment or be unexported
server/gnocco/config.go:51:1
⚠  https://revive.run/r#exported  exported type GnoccoServer should have comment or be unexported
server/gnocco/server.go:14:6
⚠  https://revive.run/r#exported  type name will be used as gnocco.GnoccoServer by other packages, and that stutters; consider calling this Server
server/gnocco/server.go:14:6
⚠  https://revive.run/r#exported  exported method GnoccoServer.Addr should have comment or be unexported
server/gnocco/server.go:23:1
⚠  https://revive.run/r#exported  exported method GnoccoServer.Logger should have comment or be unexported
server/gnocco/server.go:27:1
⚠  https://revive.run/r#exported  exported method GnoccoServer.DumpCache should have comment or be unexported
server/gnocco/server.go:31:1
⚠  https://revive.run/r#exported  exported method GnoccoServer.Run should have comment or be unexported
server/gnocco/server.go:59:1
⚠  https://revive.run/r#exported  exported method GnoccoServer.ShutDown should have comment or be unexported
server/gnocco/server.go:106:1
⚠ 9 problems (0 errors, 9 warnings)

Warnings:
8  exported
1  package-comments
```
Please create a PR for that

### PR DESCRIPTION
**Pull request by amery**
fixes #7

### CODE REVIEW

File: server/gnocco/server.go [hunk 1]
```diff
@@ -11,6 +12,7 @@ import (
 	"github.com/miekg/dns"
 )
 
+// Server implements a cached DNS resolver
```
Comment (karasz) [inline server/gnocco/server.go:15]: I don't want to be nitpicking but if this is a Resolver, shouldn't it be called as such?
gnocco.Resolver sounds even better than gnocco.Server

File: server/gnocco/config.go [hunk 2]
```diff
@@ -48,6 +48,7 @@ type hostsCfg struct {
 	RefreshInterval uint
 }
 
+// NewFromFilename creates a new Gnocco configuration from a TOML file
```
Comment (karasz) [inline server/gnocco/config.go:51]: Can we please make this NewFromFile?

Comment (amery) [reply server/gnocco/config.go]: NewFromTOML()?

File: server/gnocco/server.go (hunk 1 above)
Comment (amery) [reply server/gnocco/server.go]: amended accordingly

**Review by karasz — APPROVED** [review]
now LGTM

**Pull request merged**